src/test/data/JsonSerializableGradPadTest/invalidModuleGradPad.json

```diff
@@ -0,0 +1,6 @@
+{
+  "modules": [ {
```

Comment:
This module is valid HAHA
It's why the test for invalid module in `JsonSerializableGradPadTest` isn't passing